LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
```

Comment:
Probably it should have content and copyright as in here
https://github.com/Gapminder/gapminder-tools-vizabi/blob/development/LICENSE